Makefile

```diff
@@ -13,3 +13,6 @@ upload:
 
 requirements-dev:
 	pip install -r requirements_dev.txt
+
+lint:
```

Comment:
This command should be run automatically within [pr_checks.yml](https://github.com/pdyba/lambdalizator/blob/master/.github/workflows/pr_checks.yml#L7) - if you only run something manually, it won't be of full help 😉